go.mod

```diff
@@ -74,6 +74,7 @@ require (
 	github.com/hashicorp/vault-plugin-auth-cf v0.0.0-20190821162840-1c2205826fee
 	github.com/hashicorp/vault-plugin-auth-gcp v0.5.2-0.20190930204802-acfd134850c2
 	github.com/hashicorp/vault-plugin-auth-jwt v0.5.2-0.20191010173058-65cf93bad3f2
+	github.com/hashicorp/vault-plugin-auth-kerberos v0.0.0-20191209193128-e7680d2ebcaa
```

Comment:
Should this be updated to [v0.1.1](https://github.com/hashicorp/vault-plugin-auth-kerberos/releases/tag/0.1.1)?

Reply:
The commit it's at is correct, but for some reason it's not pulling in v.0.1.1. It's probably that I need to not just release it on Github, but do something with Go modules. Digging into it.

Reply:
I figured it out. It was very subtle. I'd released version `0.1.1` versus `v0.1.1`. When I edited the releases to prepend a `v`, go modules was more than happy to pull in the the proper version reference here.